File: reddio-java/reddio-api/src/main/java/com/reddio/api/v1/rest/GetContractInfoMessage.java
package com.reddio.api.v1.rest;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor(staticName = "of")
public class GetContractInfoMessage {
    @JsonProperty("type")
    private String type;
    @JsonProperty("contract_address")
    private String contractAddress;
}

File: reddio-csharp/reddio.api/V1/Rest/GetContractInfoMessage.cs
using Newtonsoft.Json;

namespace Reddio.Api.V1.Rest
{
    public class GetContractInfoMessage
    {
        public GetContractInfoMessage()
        {
        }

        public GetContractInfoMessage(string type, string contractAddress)
        {
            Type = type;
            ContractAddress = contractAddress;
        }

        [JsonProperty("type")] public string Type;
        [JsonProperty("contract_address")] public string ContractAddress;
    }
}